snuba/views.py

```diff
@@ -328,7 +330,10 @@ def parse_and_run_query(dataset, request: Request, timer) -> QueryResult:
         'sample': request.query.get_sample(),
     }
 
-    return raw_query(request, query, clickhouse_ro, timer, stats)
+    with sentry_sdk.start_span(description="raw_query", op="db") as span:
+        span.set_tag("table", stats["clickhouse_table"])
```

Comment:
please let's add the dataset class name. there may be multiple datasets loading from the same table

Reply:
Also you can add the actual Clickhouse query (just like we do for postgres on sentry) by doing this 
```
span.set_tag("query", query.format_sql())
```

Or even adding the dataset name/table to the description of the span.

Reply:
Sure thing. If we have the dataset & query we don't need the table, as it is redundant.